src/html/fetch-markdown.js

```diff
@@ -97,6 +97,10 @@ async function fetch(
     };
   } catch (e) {
     if (e && e.statusCode && e.statusCode === 404) {
+      if (path.match(/^\/?404\.(md|html)$/)) {
+        // Passing status 200 since we'll first have tried 400.html with a 404
+        return safebail(logger, `Could not find Markdown at ${options.uri}`, e, 200);
```

Comment:
this is IMO wrong: you must not return a `200`. otherwise fastly doesn't know that the resource is missing and cannot fallback to static processing...which actually renders this entire feature useless :-)

Reply:
We can control the fallback, but it should definitely be a 404.